.github/workflows/test.yml

```diff
@@ -20,10 +20,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:  # https://github.com/ruby/setup-ruby/blob/master/README.md#supported-versions
-        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, jruby-9.1, jruby-9.2]
+        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, '3.0', 3.1, jruby-9.1, jruby-9.2]
```

Comment:
special case with `3.0`? it's the only one within quotes 🤔

Reply:
https://github.com/actions/runner/issues/849

Reply:
🤦 YAML...